projects/plugins/jetpack/_inc/lib/admin-pages/class-jetpack-redux-state-helper.php

```diff
@@ -168,6 +168,8 @@ public static function get_initial_state() {
 				'plan'                       => Jetpack_Plan::get(),
 				'showBackups'                => Jetpack::show_backups_ui(),
 				'showRecommendations'        => Jetpack_Recommendations::is_enabled(),
+				/** This filter is documented in my-jetpack/src/class-initializer.php */
+				'showMyJetpack'              => apply_filters( 'jetpack_my_jetpack_should_initialize', true ) && ! is_multisite(),
```

Comment:
I wanted to use `Initializer::should_initialize()` for the initial state data, but can't because it also checks `did_action( 'my_jetpack_init' )` which hasn't happened yet.

Reply:
I've pushed 6c514a8d8c70565951d7d40a48e64440a37cbafa

I think it makes sense to leave `did_action` out of `should_initialize`. They are checking different things (if it should initialize and if it is already initialized)